packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -240,7 +266,9 @@ const makeRenderer = (
 			initialNow: new Date().getTime(),
 			variants: getVariants(state),
 			entryPath: url.pathname, // the path that the user entered the app on
-		},
+			uas: headers["user-agent"],
```

Comment:
what is this used for? Also, just as an aside, `UA` is the conventional acronym for user agent string, rather than `uas`, although I think I would tend toward something more explicit like `userAgent` in this case because `state.config` doesn't offer any clues about how to interpret acronym keynames.

Reply:
It's not currently being used. Lori added it when we were pairing, I believe her reasoning was better to have more info just in case. Generally I prefer to avoid abbreviating, when we added it it didn't seem ambiguous but in hindsight I can see why it is. Tagging @eilinora so she can see the feedback/info.